src/components/table/LightTable.jsx

```diff
@@ -172,8 +186,11 @@ export class LightTable extends React.PureComponent {
             />
             <TableBody>
               {rows && rows.length > 0 ? (
-                rows.map((row) => (
-                  <TableRow key={rowKey ? rowKey(row) : row.id} className={classes.row}>
+                rows.map((row, index) => (
+                  <TableRow
+                    key={rowKey ? rowKey(row) : row.id}
+                    className={`${classes.row} ${index % 2 ? classes.oddRow : classes.evenRow}`}
```

Comment:
Rather than spacing out classnames, you should use the `clsx` class to include logic like this (checkout `SidebarDrawer.jsx` for an example)